contracts/rewards/AutoCompoundingRewards.sol

```diff
@@ -206,16 +206,16 @@ contract AutoCompoundingRewards is IAutoCompoundingRewards, ReentrancyGuardUpgra
     }
 
     /**
-     * @inheritdoc IAutoCompoundingRewards
+     * @dev creates a program for a pool
      */
-    function createProgram(
+    function _createProgram(
```

Comment:
Please move private/internal function after all external/public functions

Reply:
Sure, but the purpose was to improve the quality of the review by reducing the number of changes to a minimum.
@yudilevi : Please inform when you have completed reviewing this part.

Reply:
you can move it, it's not an issue for the review

Reply:
Done